tests/testOptionMigrationWs.php

```diff
@@ -126,12 +126,12 @@ public function testThatCorrectFieldDocsShowWhenMigrationIsOn() {
 	public function testThatChangingMigrationToOffKeepsTokenData() {
```

Comment:
Setting these test to use the same value that the validator would expect from the UI.